packs/monorail/config.yml

```diff
@@ -0,0 +1,13 @@
+name: MonoRail from newrelic.com/integration
```

Comment:
This should probably be moved under the `dotnet` umbrella folder

Reply:
moved 👍